extensions/amp-jwplayer/1.0/base-element.js

```diff
@@ -0,0 +1,29 @@
+import {BentoJwplayer} from './component';
+
+import {VideoBaseElement} from '../../amp-video/1.0/video-base-element';
+
+export class BaseElement extends VideoBaseElement {}
+
+/** @override */
+BaseElement['Component'] = BentoJwplayer;
+
+/** @override */
+BaseElement['props'] = {
+  'playerId': {attr: 'data-player-id', type: 'string'},
+  'mediaId': {attr: 'data-media-id', type: 'string'},
+  'playlistId': {attr: 'data-playlist-id', type: 'string'},
+  'contentRecency': {attr: 'data-content-recency', type: 'string'},
+  'contentBackfill': {attr: 'data-content-backfill', type: 'boolean'},
+  'adCustParams': {attr: 'data-ad-cust-params', type: 'string'},
+  'adMacros': {attrPrefix: 'data-ad-macro-'},
+  'config': {attrPrefix: 'data-config'},
```

Comment:
Should this also have an ending dash?
```suggestion
  'config': {attrPrefix: 'data-config-'},
```